crates/router/src/core/fraud_check/types.rs

```diff
@@ -123,6 +125,14 @@ pub struct FrmFulfillmentRequest {
     ///contains details of the fulfillment
     #[schema(value_type = Vec<Fulfillments>)]
     pub fulfillments: Vec<Fulfillments>,
+    //name of the tracking Company
+    #[schema(max_length = 255, example = "fedex")]
+    pub tracking_company: Option<String>,
+    //tracking ID of the product
+    #[schema(max_length = 255, example = "track_8327446667")]
+    pub tracking_numbers: Option<String>,
+    //tracking_url for tracking the product
+    pub tracking_urls: Option<String>,
```

Comment:
```suggestion
    //tracking ID of the product
    #[schema(max_length = 255, example = "track_8327446667")]
    pub tracking_number: Option<String>,
    //tracking_url for tracking the product
    pub tracking_url: Option<String>,
```

as per the description